sdk/src/main/java/io/opentelemetry/sdk/trace/TracerSdk.java

```diff
@@ -50,6 +55,9 @@
   @GuardedBy("this")
   private final List<SpanProcessor> registeredSpanProcessors = new ArrayList<>();
 
+  @GuardedBy("this")
+  private boolean isStopped = false;
```

Comment:
Use a volatile boolean instead of protecting this with the sync

Reply:
SG, done.